include/openssl/ssl.h

```diff
@@ -437,17 +437,30 @@ OPENSSL_EXPORT int SSL_write(SSL *ssl, const void *buf, int num);
 // it's own KeyUpdate message.
 #define SSL_KEY_UPDATE_NOT_REQUESTED 0
 
+// SSL_KEY_UPDATE_NONE should not be set by the user and is only used to
+// indicate that there is a pending key operation. OpenSSL indicates that -1 is
+// used, so that it will be an invalid value for the on-the-wire protocol.
```

Comment:
Is it correct to say:
```suggestion
// used, so that it will be an invalid value for the on-the-wire protocol
// when calling SSL_key_update().
```
?